core/lib/navigation-error.js

```diff
@@ -11,11 +11,18 @@ import * as i18n from '../lib/i18n/i18n.js';
 
 const UIStrings = {
   /**
-   * Warning shown in report when the page under test is an XHTML document, which Lighthouse does not directly support
-   * so we display a warning.
+   * @description Warning shown in report when the page under test is an XHTML document, which Lighthouse does not directly support so we display a warning.
```

Comment:
The `@description` shouldn't be necessary here. If there isn't an `@example` the comment becomes the description.